src/components/Explore/utils.ts

```diff
@@ -143,6 +144,8 @@ export function getMetricByChartId(chartId: string): ExploreMetric | undefined {
       return ExploreMetric.WEEKLY_CASES;
     case 'caseDensityByCases':
       return ExploreMetric.DAILY_CASES_PER_100K;
+    case 'vaccinations':
```

Comment:
Again, I think we should use a different name for the bivalent booster if we're going to rework things.